### PR TITLE
[stable/nginx-ingress] added feature flag to enable hostPort for daemonset

### PR DESCRIPTION
**Pull request by ppwfx**
pull request for feature request #872

closed #892 in favor of this as I messed up my previous fork

without providing additional flags this version behaves like version 0.3.1

### CODE REVIEW

**Comment by linki** [comment]
> It may take a few minutes for the LoadBalancer IP to be available.

Notes.txt should also be updated to reflect the case where `.Values.controller.service.enabled=false`.

File: stable/nginx-ingress/README.md [hunk 1]
```diff
@@ -55,10 +55,14 @@ Parameter | Description | Default
 `controller.scope.namespace` | namespace to watch for ingress | `""` (use the release namespace)
 `controller.extraArgs` | Additional controller container arguments | `{}`
 `controller.kind` | install as Deployment or DaemonSet | `Deployment`
+`controller.daemonSet.hostPort.http.enabled` | create a http hostPort for controller.kind 'DaemonSet' | `False`
+`controller.daemonSet.hostPort.https.enabled` | create a https hostPort for controller.kind 'DaemonSet' | `False`
+`controller.daemonSet.hostPort.healthz.enabled` create a healthz hostPort for controller.kind 'DaemonSet' | `False`
```
Comment (mgoodness) [inline stable/nginx-ingress/README.md:60]: Nit: let's keep the case consistent (`false`, not `False`)

I also think five keys deep is excessive. How about `controller.hostPort.httpEnabled`, `controller.hostPort.httpsEnabled`, and `controller.hostPort.healthzEnabled`?

File: stable/nginx-ingress/README.md [hunk 2]
```diff
@@ -55,10 +55,14 @@ Parameter | Description | Default
...
 `controller.nodeSelector` | node labels for pod assignment | `{}`
 `controller.podAnnotations` | annotations to be added to pods | `{}`
 `controller.replicaCount` | desired number of controller pods | `1`
 `controller.resources` | controller pod resource requests & limits | `{}`
+`controller.service.enabled` | enable controller service | `True`
```
Comment (mgoodness) [inline stable/nginx-ingress/README.md:65]: Nit: see above

**Review by mgoodness — CHANGES_REQUESTED** [review]
A couple nits, one usability

**Comment by mgoodness** [comment]
Closes #872

**Comment by unguiculus** [comment]
Marking this as stale. Please update during the next week.

**Comment by viglesiasce** [comment]
Closing as stale.

**Pull request closed without merging**